### PR TITLE
Add translation messages

### PR DESCRIPTION
**Pull request by milesj**
This is a test/work in progress. I wanted to see how easily we could replace all strings with a translate function, and the ability to overwrite all translations globally.

As it currently stands, I believe a translation layer like this is necessary, as strings like "yes" and "no" are very reliant on the user's natural language. It would be best to localize those.

Besides this approach, another option we can look into is being able to customize the messages per question object passed in.

### CODE REVIEW

**Comment by milesj** [comment]
Sounds good to me. I'll make a few changes.

**Comment by milesj** [comment]
@terkelg Moved the functionality into a `translate` util function. Thoughts?

**Comment by terkelg** [comment]
Thanks @milesj. Second thoughts: Why don't we just expose these as options in each prompt constructor? Wouldn't that be even simpler? I'm just curious how much this is going to be used

**Comment by milesj** [comment]
@terkelg Sounds good, will give that a try.

**Comment by terkelg** [comment]
Yeah looks good! I'll take some time this weekend to go trough PRs 👍

File: lib/elements/confirm.js [hunk 1]
```diff
@@ -66,8 +69,8 @@ class ConfirmPrompt extends Prompt {
           color.bold(this.msg),
           style.delimiter(this.done),
           this.done
-            ? this.value ? 'yes' : 'no'
-            : color.gray(this.initialValue ? '(Y/n)' : '(y/N)')
+            ? this.value ? yesLabel || 'yes' : noLabel || 'no'
```
Comment (lukeed) [inline lib/elements/confirm.js]: Do you think it'd be better to move these defaults into the constructor & on the `opts` object directly? In my opinion this is a little hard to read, but certainly doable

Comment (terkelg) [reply lib/elements/confirm.js]: Agree

Comment (lukeed) [reply lib/elements/confirm.js]:
```js
const { labels } = this.opts;

//

this.done
  ? this.value ? labels.yes : labels.no
  : color.gray(this.initialValue ? labels.defaultYes : labels.defaultNo)
```

**Comment by milesj** [comment]
@terkelg @lukeed Tried another approach for confirm, let me know what you think.

**Comment by terkelg** [comment]
👍Waiting for @lukeed to confirm

**Comment by terkelg** [comment]
Yeah, good point @lukeed. It's good for now tho, thanks @milesj. Do you mind adding the new options to the docs? We're getting there

**Comment by milesj** [comment]
Yeah can do. I think it may be best to just handle non-latin based languages right now, so I'll do both.

**Comment by milesj** [comment]
Done.

**Pull request merged**